doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -21,6 +21,24 @@
 <a name="3.1"></a>
 ## 3.1 Introduction
 
+The CNTT Kubernetes Reference Architecture (RA) aims to provide an industry standard reference architecture independent of the many Kubernetes offerings and distributions. The RA does not seek to require vendor-specific enhancements in order to achieve CNTT compliance; compliance is achieved using upstream components or features that are developed by the open source community. This would allow operators to provide a common Kubernetes-based architecture that allows any compliant VNF or CNF to be deployed and operated as expected. The purpose of this chapter is to outline all the components required to provide Kubernetes in a consistent and reliable way.
+
+Kubernetes is already very well documented at [https://kubernetes.io/docs/home/](https://kubernetes.io/docs/home/) so rather than repeat content from there this and following chapters will describe the specific features used and how we expect them to be implemented.
+
+This reference architecture provides optionality in terms of pluggable components such as service mesh and other plugins that might be used, however the focus of the reference architecture is on the abstracted interfaces and features that are required for workload management and execution.
+
+Chapter 5 of the Reference Model (RM) describes the [hardware](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#5.4) and [software](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter05.md#52-nfvi-sw-profiles-features-and-requirements) profiles, which are descriptions of the capabilities and features that the NFVI offer to the workloads. As of v2.0, Figure 5-3 in the RM (also shown below) depicts a high level view of the software profile features that apply to each instance profile (Basic, Network Intensive, Compute Intensive). For more information on the instance profiles please read [RM Chapter 4, section 4.2.4](https://cntt-n.github.io/CNTT/doc/ref_model/chapters/chapter04.html#4.2.4).
+
+<p align="center"><img src="../../../ref_model/figures/RM_chap5_fig_5_3_SW_profile.png" width="80%"/></p>
+<p align="center"><b>Figure 5-3 (from RM):</b> NFVI software profiles</p>
+
+In addition, Figure 5-4 (also shown below) depicts the hardware profile features that apply to each instance profile.
+
+<p align="center"><img src="../../../ref_model/figures/RM_chap5_fig_5_4_HW_profile.png" width="80%"/></p>
+<p align="center"><b>Figure 5-4 (from RM):</b> NFVI hardware profiles and host associated capabilities</p>
+
+These features and capabilities that are described in the software and hardware profiles are considered throughout this RA, with traceability of the RA requirements to the RM requirements formally captured in [chapter 2, section 2.2](https://cntt-n.github.io/CNTT/doc/ref_arch/kubernetes/chapters/chapter02.html#2.2) of this RA.
```

Comment:
Fixed

Reply:
A question: including a reference to RM makes it look like everything described there is applicable for RA2. I'm wodering how to handle things like CPU pinning, vswitch optimization, security group support, etc. These things make sense in an OpenStack context, not so much in K8s...

Reply:
Yeah I agree. I was going to chat to @markshostak first, but I was thinking of creating an Issue to address this - the solution being either to make the RM more agnostic, or to expand the RM to address both VM and container-based workloads with an additional "column" showing which is applicable to each.

Reply:
I think the problem is two-folded:
- there are things in RM which have a very OpenStack specific name (but otherwise as generalized functions they are applicable in K8s as well) - here the answer is to do the generalization in the RM text

- there are things that were useful an applicable in the OpenSTack context (like CPU pinning), but I'm not even sure if we should allow that capability in RA2 in the cloud native context. So updating the RM is one thing, but perhaps we need a discussion on the platform capabilities that we are not so much fan of any more when going cloud native. I think CPU pinning, SR-IOV are some examples.

Reply:
I see - yes understood. For the second point I think it's linked to the discussion in #699. Specifically what Rabi is potentially going to document: https://github.com/cntt-n/CNTT/pull/699#issuecomment-559540571

Reply:
Have added the following in this section:

> As this RA is developed, it is quite possible that modifications to the RM are requested in order to make the RM more technology agnostic (i.e. to cater for VMs and Containers) or to add container-specific specifications to the RM.

Reply:
I have #621 up to make the RM more agnostic, so if you find something what is OpenStack specific please comment it there. 
Also based on the discussions in #621 I plan to create a pr where I add a compatibility matrix against the RM to RA2.